src/customers/account-dashboard-gift-registry.md

```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
 
 ![]({% link images/images-ee/account-dashboard-gift-registry.png %}){: .zoom}
 _Gift Registry_
+
+## To create new gift registry
+
+1. From customer account dashboard, go to **Gift Registry**.
```

Comment:
1. Select **Gift Registry** in the customer account dashboard.

Is that a clickable option **Gift Registry**?

Reply:
Yes, this is a clickable option but not a button 😊

I have implemented this suggestion!